daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -50,34 +24,65 @@ Parameter | Description
 `workflowName` | Identify the workflow type
 `instanceId` | Unique value created for each run of a specific workflow
 
+### Request Contents
 
-### Headers
+In the request you can pass along headers:
 
-As part of the start HTTP request, the caller can optionally include one or more `dapr-workflow-metadata` HTTP request headers. The format of the header value is a list of `{key}={value}` values, similar to the format for HTTP cookie request headers. These key/value pairs are saved in the workflow instance’s metadata and can be made available for search (in cases where the workflow implementation supports this kind of search).
-
-
-## HTTP responses
+```json
+{
+  "input": {
+    // arguments to pass to the workflow
```

Comment:
Calling it "arguments" (plural) isn't quite correct since workflows can only take a single argument. Also, the argument doesn't necessarily have to be a JSON object, which seems to be implied by the `{...}` here. It can be any valid JSON data, including objects, strings, numbers, or arrays.

Reply:
Updated with a proposal based on what you shared - Thoughts?

Reply:
Yeah, I like the update you provided.